specification/cognitiveservices/OpenAI.Inference/models/images.tsp

```diff
@@ -0,0 +1,122 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/http";
+
+using Azure.Core;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("The desired size of the generated images. Must be one of 256x256, 512x512, or 1024x1024.")
+enum ImageSize {
+    `256x256`,
+    `512x512`,
+    `1024x1024`,
+}
+
+@doc("Defines the request to create an operation to generate images.")
+model ImageRequest {
+    @doc("A text description of the desired image(s).")
+    prompt: string;
+
+    @doc("The number of images to generate.")
+    n: int32;
+
+    @doc("The desired size of the generated images. Must be one of 256x256, 512x512, or 1024x1024.")
+    size: ImageSize;
```

Comment:
optional, defaults to 1024x1024

Reply:
made optional and added the default to the doc